coral/src/app/components/Form.test.tsx

```diff
@@ -16,7 +16,6 @@ import {
 } from "src/app/components/Form";
 import { renderForm } from "src/services/test-utils/render-form";
 import { z } from "zod";
-import { waitForElementToBeRemoved } from "@testing-library/react/pure";
```

Comment:
Just the fact that we could remove that from the test is kind of a good sign, isn't it? :D

Reply:
Yes, the error is removed immediately on change, so the `waitForElementToBeRemoved` was waiting for something to be removed that was already removed haha